src/static/js/home.js

```diff
@@ -0,0 +1,15 @@
+const menu = document.querySelector(".menu");
+const menuIcon = document.querySelector(".menu-icon");
+const menuCloseIcon = document.querySelector(".menu-close-icon");
+
+menuIcon.addEventListener('click', () => {
+  menu.style.display = "flex";
```

Comment:
Rather than changing the specific styles in JS, let's toggle a "hidden" class on the necessary elements, which could be a generic CSS rule to `display: none`. Removing the class should revert it back to its original display value, so the JS doesn't need to care whether it was flex or block or inline etc.